scenes/enemy-commander/EnemyCommander.gd

```diff
@@ -0,0 +1,39 @@
+tool
```

Comment:
Do we need this script to be a tool? I think we can remove it but I'll wait for your confirmation about this!

Reply:
Sorry, I used it to do some test locally and forget to remove :blush: